packages/semcom-node/lib/store/services/component-pod-store.service.spec.ts

```diff
@@ -0,0 +1,215 @@
+import fetch from 'node-fetch';
+import { Quad, DataFactory } from 'n3';
+import { default as streamify } from 'streamify-array';
+import serialize from 'rdf-serialize';
+import { ComponentMetadata, LoggerConsoleService } from '@digita-ai/semcom-core';
+import { ComponentTransformerService } from '../../component/services/component-transformer.service';
+import { ComponentPodStore } from './component-pod-store.service';
+
+const { namedNode, quad: createQuad } = DataFactory;
+
+jest.mock('node-fetch');
+
+const { Response } = jest.requireActual('node-fetch');
+
+describe('ComponentPodStore', () => {
+
+  const podUri = 'http://localhost:9000/metadata/';
+
+  const quadTransformer: ComponentTransformerService = new ComponentTransformerService(new LoggerConsoleService());
+
+  const componentPodStore = new ComponentPodStore(podUri, quadTransformer);
+
+  let fetchMock: jest.MockedFunction<typeof fetch>;
+
+  interface ComponentMetadataMock {
+    component: ComponentMetadata;
+    filename: string;
+  }
+
+  const streamToBuffer = (stream: NodeJS.ReadableStream): Promise<string> => new Promise((resolve) => {
+
+    const bufs = [];
+    stream.on('data', (chunk) => bufs.push(chunk));
+    stream.on('end', () => resolve(bufs.join('')));
```

Comment:
```suggestion
    stream.on('end', () => resolve(Buffer.concat(buf)));
```
Buffer has it's own method to concatenate an array of Buffers

Reply:
I meant to have the method be called streamToString - i changed it